app/controllers/contributions_controller.rb

```diff
@@ -52,9 +52,9 @@ def contribution_blueprint_options
   end
 
   def filter_params
-    return {} unless allowed_params&.to_h.any?
+    return {} unless allowed_params.to_h.any?
```

Comment:
I don't think `allowed_params` can be nil?

Reply:
Right. It can be empty/blank or it can raise an error, but it can't be `nil`